accounts-db/src/accounts_db.rs

```diff
@@ -6246,18 +6269,23 @@ impl AccountsDb {
             &HashSet::default(),
         );
 
+        let mut store_accounts_timing = StoreAccountsTiming::default();
+        let mut store_elapsed_us = 0;
         if !is_dead_slot {
             // This ensures that all updates are written to an AppendVec, before any
             // updates to the index happen, so anybody that sees a real entry in the index,
             // will be able to find the account in storage
             let flushed_store = self.create_and_insert_store(slot, total_size, "flush_slot_cache");
-            self.store_accounts_frozen(
-                (slot, &accounts[..]),
-                Some(hashes),
-                &flushed_store,
-                None,
-                StoreReclaims::Default,
-            );
+            let (store_accounts_timing_inner, store_elapsed_inner_us) = measure_us!(self
```

Comment:
Would it be better to move `store_elapsed_inner_us` into `StoreAccountsTiming` struct?

Reply:
could be done, just messy.

Reply:
OK. We can leave it to the future.